Dockerfile.shapes32

```diff
@@ -0,0 +1,30 @@
+FROM ubuntu:xenial
+
+RUN dpkg --add-architecture i386 && apt-get update && apt-get install -y \
+    curl \
+    g++ \
+    make \
+    perl-base \
+    perl-modules \
+    lib32gcc-5-dev \
+    lib32stdc++-5-dev \
+    libboost1.58-dev \
+    libc6-dev-i386 \
```

Comment:
Looks like this shouldn't be needed.  libc6-dev-i386 provides libraries in /usr/lib32, which shouldn't be used.  libc6-dev:i386, on the other hand, provides libraries in /usr/lib/i386-linux-gnu, which should be used.